charts/castai-cluster-controller/values.yaml

```diff
@@ -118,7 +118,13 @@ affinity:
 
 # Pod security context.
 # Ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
-securityContext: {}
+# https://github.com/GoogleContainerTools/distroless/blob/530158861eebdbbf149f7e7e67bfe45eb433a35c/common/variables.bzl#L18
+# securityContext -- User 65532 is non-root user for gcr distorless images
+securityContext:
+  runAsNonRoot: true
```

Comment:
Does chart_upsert action work?

Reply:
Yes, it was tested with adding evictor to the cluster, and cluster-controller was still able to apply the changes.